### PR TITLE
phi0472.phi001

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated headers to reflect First1K standards.

Also @lcerrato I have a question for you. The two headers for the English translations that are from the same edition have two different ways of showing responsibility in the monogr section. 

phi0472.phi001.perseus-eng3 has:
   <respStmt>
        <name>Sir Richard Francis Burton</name>
        <resp>translator</resp>
      </respStmt>

while phi0472.phi001.perseus-eng4 has:
 <editor role="editor">Leonard C. Smithers</editor>    

I wasn't sure what one should be used. Thanks!

### CODE REVIEW

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5343)*

File: data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
     <fileDesc>
       <titleStmt>
         <!--This is the overall file title. It's not really used.-->
-        <title>Carmina</title>
+        <title xml:lang="lat">Carmina</title>
         <title type="sub">Machine readable text</title>
-        <author>C. Valerius Catullus</author>
+        <author xml:lang="lat">C. Valerius Catullus</author>
         <editor role="editor" n="Burton">Sir Richard Francis Burton</editor>
```
Comment (lcerrato) [inline data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml:11]: This is wrong. Generally translations need a translator, not an editor. So it is `editor role="translator`" and the n attribute is not needed. This is one of those things that jump out at you when work on lots of headers as the old files are completely inconsistent on this.

File: data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml [hunk 2]
```diff
@@ -31,18 +31,18 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
   <biblStruct>
     <monogr>
       <author>Catullus</author>
-      <title>Carmina</title><idno type="OCLC">878062</idno>
+      <title>The Carmina of Gaius Valerius Catullus.</title>
       <respStmt>
```
Comment (lcerrato) [inline data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml:35]: We don't generally use the `<respStmt>` here anymore. We use `<author> <title> <editor>`
If the editor is not clear, it need not appear.

You could use <author> and <title> with the title being the longer title here. (The short title is fine for the CTS metadata.)

But we definitely don't use the <respStmt> here

**Comment by lcerrato** [comment]
I see that the Smithers text lists Smithers as the editor of the edition, and the Burton text lists Burton as the translator.

The monogr editor is whoever is listed on the frontplate at the editor, so it could be Burton, both, or neither.
https://catalog.hathitrust.org/Record/007936727

I would probably credit both as editors of the volume in the monogr section and translators of the work at the start of the header.

It looks like the full title is The Carmina of Caius Valerius Catullus. 
Now first completely Englished into verse and prose, the metrical part by Captain Sir Richard F. Burton, and the prose portion, introduction and notes explanatory and illustrative by Leonard C. Smithers.

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml [hunk 3]
```diff
@@ -4,9 +4,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
      <teiHeader>
           <fileDesc>
                <titleStmt>
-                    <title>Carmina</title>
+                    <title xml:lang="lat">Carmina</title>
                     <title type="sub">Machine readable text</title>
```
Comment (lcerrato) [inline data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml:8]: I just noticed this — we've been removing the machine readable text. (There are just so many things....)

File: data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml [hunk 4]
```diff
@@ -4,9 +4,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
...
-                    <author n="Catul.">C. Valerius Catullus</author>
+                    <author xml:lang="lat">C. Valerius Catullus</author>
                     <editor role="editor" n="Smithers">Leonard C. Smithers</editor>
```
Comment (lcerrato) [inline data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml:10]: See my comment on the other file. I have no idea why `<editor role="editor">` was ever a practice.

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml:35]: Hey @lcerrato thanks for all the comments. I think I've made all the fixes but have one last question by long title do you mean the whole title above:
"The Carmina of Gaius Valerius Catullus. Now first completely Englished into verse and prose, the metrical part by Captain Sir Richard F. Burton, and the prose portion, introduction and notes explanatory and illustrative by Leonard C. Smithers." or just "The Carmina of Gaius Valerius Catullus". Thanks!

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5346)*

File: data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml (hunk 2 above)
Comment (lcerrato) [reply data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml]: I mean the whole thing, since that captures the two individuals involved.

Comment (AlisonBabeu) [reply data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml]: OK I will do that, fix the files and update it one more time.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5348)*

**Review by lcerrato — APPROVED** [review]

**Pull request merged**